.github/workflows/ci-tests.yml

```diff
@@ -15,6 +15,7 @@ jobs:
     defaults:
```

Comment:
You would want to add `path` to `on: push:` to run the workflow only when files in `python` are updated

Reply:
Do you think we need on `push` if we also run this on `pull_request`?